library/general/src/modules/Report.rb

```diff
@@ -605,7 +605,9 @@ def LongError(error_string, width: 60, height: 10)
       Builtins.y2error(1, "%1", error_string) if @log_errors
 
       if @display_errors
-        if Ops.greater_than(@timeout_errors, 0)
+        if Mode.commandline
```

Comment:
Well, when you touch it, is possible to fix also longWarning and LongInfo?

Reply:
Done in #901 .